dags/pull_weather.py

```diff
@@ -0,0 +1,92 @@
+"""
+Pipeline to pull weather prediction data from Envrionment Canada and upsert into weather.prediction_daily table.
+Then, 
+A Slack notification is raised when the airflow process fails.
+"""
+import sys
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.models import Variable
+from dags.pull_wys import SLACK_CONN_ID 
+
+#connection credentials
+ec2_cred = PostgresHook("weather_bot")
+
+#import python scripts
+try:
+    sys.path.append('script-path')
```

Comment:
you might need to replace this line with the following lines to avoid hardcoding any paths:
```
repo_path = os.path.abspath(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
sys.path.insert(0,os.path.join(repo_path,'weather'))
```
Also, don't forget to `import os`